packages/dds/tree/docs/wip/TreeStorage.md

```diff
@@ -0,0 +1,496 @@
+# Tree Storage
```

Comment:
@dstanesc - Craig has been leading the storage design for SharedTree.  It's easiest to read his document if you push the
<img width="39" alt="image" src="https://user-images.githubusercontent.com/11368662/172478866-0f5f0028-e18f-4e8e-bf3d-91bf4b40507b.png">button.  (It's on the right in the gray bar just above this file.)

@CraigMacomber - Dragos has independently been doing some experiments with storage organization.  You can find them here: https://github.com/dstanesc/graph-playground